### DISCUSSION TITLE
Auto install dependencies on `./bin/kamal setup`

### DISCUSSION THREAD

**Discussion by Kaakati** [basecamp/kamal#677] [Ideas]
Since Kamal is able to SSH to the servers, I think it should automatically configure its requierments.
Running the `setup` command should install dependencies such as running
`sudo apt install -y docker.io curl git` & add the $USER to docker group & since sometimes it does not get added to run the `newgrp docker` and then proceeds with the process.

**Comment by acidtib** [comment]
`kamal server bootstrap` will bootstrap the servers

if you are not using user `root` then you need to manually bootstrap the server your self 

https://kamal-deploy.org/docs/configuration#using-a-different-ssh-user-than-root
